common/redshift/Makefile

```diff
@@ -88,7 +88,7 @@ clean-deploy: check
 	$(MAKE) share-remove
 	$(MAKE) schema-remove || ((sleep 5 && $(MAKE) schema-remove) || exit 1)
 
-storage-upload: build venv3
+storage-upload: build venv2 venv3
```

Comment:
I think venv3 is not needed here.

Reply:
You are right, I just removed it and it works locally. It's weird though since previously it was mostly required for having wheel installed in the environment. Let's see if it works in the PR.